mathesar_ui/src/components/cell-fabric/data-types/string.ts

```diff
@@ -44,6 +44,7 @@ const stringType: CellComponentFactory = {
       },
     };
   },
+  getDisplayFormatter: () => String,
```

Comment:
We need to quote strings which have `\t` characters. This applies for all types.